tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/AWS/LambdaCommon.cs

```diff
@@ -100,9 +93,14 @@ internal static bool SendStartInvocation(ILambdaExtensionRequest requestBuilder,
             return ValidateOKStatus((HttpWebResponse)request.GetResponse());
         }
 
-        internal static bool SendEndInvocation(ILambdaExtensionRequest requestBuilder, bool isError)
+        internal static bool SendEndInvocation(ILambdaExtensionRequest requestBuilder, bool isError, string data)
         {
             var request = requestBuilder.GetEndInvocationRequest(isError);
+            var byteArray = Encoding.UTF8.GetBytes(data ?? DefaultJson);
```

Comment:
I may be missing something but I don't think you need the null check here

```suggestion
            var byteArray = Encoding.UTF8.GetBytes(data);
```